### ISSUE TITLE
Feature request: Definition lists

### ISSUE THREAD

**Issue by singularitti** [typora/typora-issues#1523]
Typora is a very good tool to organize notes for scientists since it supports MathJax. However, there will be sometimes where those people write theorems, propositions, lemmas, definitions, etc. If there will be a syntax for writing these, that will be super-great for those people. [StackEdit](https://stackedit.io/app#) has implemented such syntax and it is [open-source](https://github.com/benweet/stackedit):

```markdown
Markdown
:  Text-to-HTML conversion tool
```

and will render like this
<img width="712" alt="image" src="https://user-images.githubusercontent.com/25192197/40634499-55540672-62c3-11e8-9615-82447503e4de.png">

Hope Typora can provide such syntax, thanks!

**Comment by rose00** [comment]
+1 the definition pattern is common in the Java specs also

The name of this feature in Pandoc's terminology for extensions is `definition_lists`

In general, Pandoc's default set of extensions is very useful, and probably provides a "sweet spot" of default (or optional) features for Typora.

Other Pandoc extensions I use and would like to see in Typora are:
`shortcut_reference_links`: syntax like `[foo]` as short for `[foo][]`
`fenced_divs`: fences (with attributes) for `<div>` elements like `:::: {#id .class}`
`bracketed_spans`: link-like brackets can be attributed like `[my stuff]{.mystyle}`

Of course this assumes that Typora would do something with the style, etc., which is probably a large project.

**Comment by unknownterritory** [comment]
All features of Markdown's basic and extended syntaxes should be supported by Typora. Definition lists are part of the [extended syntax](https://www.markdownguide.org/extended-syntax/#definition-lists). Other elements belonging to the extended syntax, like strikethrough or footnotes, are already supported by Typora. I look forward to the addition of definition lists and the completion of Markdown's syntax support.

**Comment by Redcxx** [comment]
I would love to see this feature as well.

**Comment by c5creative** [comment]
Ditto!

**Comment by mayank-m2** [comment]
Will love to see this feature as well.

**Comment by volkan-a** [comment]
Looking forward for this feature.

**Comment by yifengd** [comment]
I also agree that pandoc's default extensions are a good feature set for typora. Typora already uses pandoc for rendering anyways.

**Comment by donaldsoncd** [comment]
Also hoping this becomes a part of Typora!

**Comment by triangulum** [comment]
Found my way here through the search engines too. Definition lists would be very welcome!

**Comment by MauvaisJoueur** [comment]
I'd welcome this feature that is the only formatting feature I'm missing from Typora. This feature is a must have for every type of user, not only for scientists!

**Comment by bobhy** [comment]
+1 for the feature request.

Has any decision been made about it?

**Comment by markgraf** [comment]
Definiton lists have been around since html 1.0 and are supported in quite a few markdown-dialects.
Please add support for definition lists.  They are unfortunately way too underused and people keep using semantically incorrect markup to simulate them.

**Comment by xiaotianxt** [comment]
+1 for the feature request

**Comment by alexgoussev** [comment]
Any updates?

**Comment by michael-booker** [comment]
+1 for the feature request

**Comment by alexgoussev** [comment]
Is there any news on this?

**Comment by CX330Blake** [comment]
+1, please I do need this

**Comment by ma-joel** [comment]
+1， please do it!

**Comment by Baerbeisser** [comment]
| definition | item |
| ---------- | ---- |
| **definition** | item |
| **definition** | item |

(without table) is seen pretty often in documents., where the **definition** is often bold.  
I wonder, why most lightweight markup languages are missing this - and if not, do some niche thing like this:

| definition |       |
| ---------- | ----- |
|            | item1 |
|            | item2 |

Btw, aside from themes, translation and support repos, Typora seems dead for years now, so new features are unlikely.
